src/main/java/uk/gov/ch/developer/docs/models/nav/NavBarItem.java

```diff
@@ -4,56 +4,116 @@
 import java.util.Collections;
 import java.util.List;
 
+/**
+ * Basic implementation of the INavBarItem.
+ */
 public class NavBarItem implements INavBarItem {
 
+    private final String DEFAULTURL = System.getProperties().getProperty("home.url");
```

Comment:
Sonar seems to want this camel cased however I don't know that, that would be typical of what is basically a constant. Opinions?

Reply:
it's reasonable of Sonar, as it is a non-constant function result.